### PR TITLE
Add ccf-what extension

### PR DESCRIPTION
**Pull request by CrackedPoly**
## Description

Loop up the rankings of China Computer Federation (CCF) recommended publications in Raycast.

## Screencast

screenshot

screenshot

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by CrackedPoly** [comment]
Researchers can look up a paper's publication in the China Computer Federation (CCF) recommended list, where they highlight some high-quality publications.

A similar extension would be [show-rank](https://github.com/hnshhslsh/show-rank) in Chrome.

File: extensions/ccf-what/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        detail={
+          <List.Item.Detail
+            metadata={
+              <List.Item.Detail.Metadata>
+                <List.Item.Detail.Metadata.Label title="Abbreviation" text={props.abbr} />
+                <List.Item.Detail.Metadata.Label title="Name" text={name} />
+                <List.Item.Detail.Metadata.Separator />
+                <List.Item.Detail.Metadata.Label title="Tier" icon={tier_icon} />
+                <List.Item.Detail.Metadata.Label title="Category" text={category.english} />
+                <List.Item.Detail.Metadata.Label title="分类" text={category.chinese} />
```
Comment (pernielsentikaer) [inline extensions/ccf-what/src/index.tsx:66]: At the moment, Raycast doesn't support localization and only supports US English.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Make `Search in DBLP` the primary action instead

- Add icon to `Toggle Details 

- What about using the default filtering built-in Raycast which searches title / subtitle and keywords array

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by CrackedPoly** [comment]
> Hi there, I have new updates regarding your comments and new features:
> 
> * Accepted your advice: `Search in DBLP`, `Toggle Details`, and built-in filtering
> * Flexible data source URL and update policy: set in the preference
> * Frequency sorted
> * Localization: set in the preference
>   Thanks in advance for your work and any feedback! 🔥

And some minor icon issues.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**